File: server-aspnet-fastendpoints/src/Org.OpenAPITools/Models/ShipmentOrder.cs
namespace Org.OpenAPITools.Models;


/// <summary>
/// 
/// </summary>
public class ShipmentOrder 
{
    public bool Valid { get; set; }
    public int BatchID { get; set; }
    public int BatchSize { get; set; }
    public double OrderTrackingID { get; set; }
    public string ClientRefNo { get; set; }
    public string ClientRefNo2 { get; set; }
    public string HubCode { get; set; }
    public string AccountNo { get; set; }
    public string CompanyName { get; set; }
    public string VehicleTypeName { get; set; }
    public string ServiceTypeName { get; set; }
    public string ServiceCode { get; set; }
    public int Pieces { get; set; }
    public double Weight { get; set; }
    public double CubicFeet { get; set; }
    public string PickupCompany { get; set; }
    public string PickupContact { get; set; }
    public string PickupPhone { get; set; }
    public string PickupStreet { get; set; }
    public string PickupStreet2 { get; set; }
    public string PickupCity { get; set; }
    public string PickupState { get; set; }
    public string PickupZip { get; set; }
    public DateTime PickupTargetFrom { get; set; }
    public DateTime PickupTargetTo { get; set; }
    public string DeliveryCompany { get; set; }
    public string DeliveryContact { get; set; }
    public string DeliveryPhone { get; set; }
    public string DeliveryStreet { get; set; }
    public string DeliveryStreet2 { get; set; }
    public string DeliveryCity { get; set; }
    public string DeliveryState { get; set; }
    public string DeliveryZip { get; set; }
    public DateTime DeliveryTargetFrom { get; set; }
    public DateTime DeliveryTargetTo { get; set; }
    public List<OrderPackage> OrderPackages { get; set; }
    public int RouteID { get; set; }
    public int DriverID { get; set; }
    public int PickupSequenceNo { get; set; }
    public int DeliverySequenceNo { get; set; }
    public string PickupETA { get; set; }
    public string DeliveryETA { get; set; }
}

File: server-scala-play-server/app/model/ShipmentOrder.scala
package model

import play.api.libs.json._
import java.time.OffsetDateTime

/**
  * Represents the Swagger definition for ShipmentOrder.
  */
@javax.annotation.Generated(value = Array("org.openapitools.codegen.languages.ScalaPlayFrameworkServerCodegen"), date = "2026-02-09T20:57:39.376804970Z[Etc/UTC]", comments = "Generator version: 7.20.0-SNAPSHOT")
case class ShipmentOrder(
  valid: Option[Boolean],
  batchID: Option[Int],
  batchSize: Option[Int],
  orderTrackingID: Option[Double],
  clientRefNo: Option[String],
  clientRefNo2: Option[String],
  hubCode: Option[String],
  accountNo: Option[String],
  companyName: Option[String],
  vehicleTypeName: Option[String],
  serviceTypeName: Option[String],
  serviceCode: Option[String],
  pieces: Option[Int],
  weight: Option[Double],
  cubicFeet: Option[Double],
  pickupCompany: Option[String],
  pickupContact: Option[String],
  pickupPhone: Option[String],
  pickupStreet: Option[String],
  pickupStreet2: Option[String],
  pickupCity: Option[String],
  pickupState: Option[String],
  pickupZip: Option[String],
  pickupTargetFrom: Option[OffsetDateTime],
  pickupTargetTo: Option[OffsetDateTime],
  deliveryCompany: Option[String],
  deliveryContact: Option[String],
  deliveryPhone: Option[String],
  deliveryStreet: Option[String],
  deliveryStreet2: Option[String],
  deliveryCity: Option[String],
  deliveryState: Option[String],
  deliveryZip: Option[String],
  deliveryTargetFrom: Option[OffsetDateTime],
  deliveryTargetTo: Option[OffsetDateTime],
  orderPackages: Option[List[OrderPackage]],
  routeID: Option[Int],
  driverID: Option[Int],
  pickupSequenceNo: Option[Int],
  deliverySequenceNo: Option[Int],
  pickupETA: Option[String],
  deliveryETA: Option[String]
)

object ShipmentOrder {
  implicit lazy val shipmentOrderJsonFormat: Format[ShipmentOrder] = Json.format[ShipmentOrder]
}

